src/odemis/driver/tmcm.py

```diff
@@ -875,6 +888,34 @@ def SendInstruction(self, n, typ=0, mot=0, val=0):
 
                 return rval
 
+    def _tryRecover(self):
+        self.state._set_value(HwError("TMCM disconnected"), force_write=True)
```

Comment:
As it's the state of the TMCM component it's probably not so useful to indicate "TMCM". Probably for the user, it would be more useful to indicate something like "USB connection lost" (so that he knows it's an issue with the cable or hub).